test/DynamoCoreWpfTests/WorkspaceSaving.cs

```diff
@@ -539,6 +539,11 @@ public void CanSaveAndReadWorkspaceName()
             // get file path and name of file
             var filePath = GetNewFileNameOnTempPath("dyn");
             var fileName = Path.GetFileName(filePath);
+            string extension = Path.GetExtension(filePath);
+            if (extension == ".dyn" || extension == ".dyf")
+            {
+              fileName = Path.GetFileNameWithoutExtension(filePath);
```

Comment:
Is it a must do here? I though `ViewModel.SaveAs()` will call `WorkspaceViewModel.Save()` to include this check?

Reply:
This check here does not have anything to do with the file saving, other than removing the extension from the filename (if it is either ".dyn" or ".dyf") before storing the value into the Name parameter in the JSON.

Reply:
@ColinDayOrg Ah, I see, for comparison